libs/x-content/src/main/java/org/opensearch/common/xcontent/XContentContraints.java

```diff
@@ -32,4 +33,6 @@ public interface XContentContraints {
     final int DEFAULT_MAX_DEPTH = Integer.parseInt(
         System.getProperty(DEFAULT_MAX_DEPTH_PROPERTY, "1000" /* StreamReadConstraints.DEFAULT_MAX_DEPTH */)
     );
+
+    final int DEFAULT_CODEPOINT_LIMIT = Integer.parseInt(System.getProperty(DEFAULT_CODEPOINT_LIMIT_PROPERTY, "52428800" /* ~50 Mb */));
```

Comment:
Is the implicit default currently ~3MB and with this change we are increasing it?

Reply:
> Is the implicit default currently ~3MB and with this change we are increasing it?

Correct, the implicit default is 3Mb

Reply:
@andrross any concerns with this change (I think 3mb is/was tool low for OpenSearch) ? thank you!

Reply:
No concerns! Was just clarifying.